go.mod

```diff
@@ -3,24 +3,72 @@ module github.com/redhat-developer/gitops-operator
 go 1.16
 
 require (
-	github.com/argoproj-labs/argocd-operator v0.0.16-0.20210722160114-5fe7ef0c459f
+	github.com/argoproj-labs/argocd-operator v0.0.16-0.20210802081729-d4abfb1eb224
```

Comment:
It should pick up post sdk operator upgrade tag (probably should pick up the latest).   The changes in this file below are likely not needed/different.

Reply:
rebased fixed the issue.